accesslog/middleware_gin_test.go

```diff
@@ -131,7 +131,7 @@ func TestMiddleware(t *testing.T) {
 			// First three entries in the trace should match this:
 			`trace=".+middleware_gin_test\.go\(TestMiddleware\.func[0-9]*\):[0-9]+\\n` +
 				`.+\(\(\*Context\).Next\):[0-9]+\\n` +
-				`.+middleware_gin\.go\(Middleware\.func[0-9]*\):[0-9]+\\n.+"`,
+				`.+\(\(\*Context\).Next\):[0-9]+\\n`,
```

Comment:
ok that one I do not understand. what does it have to do with `monitor` add-on?

Reply:
no, it's broken because of a different version of golang (inherited from mendertesting, we use the defaults).
Master is also failing (I ran it just to be sure: https://gitlab.com/Northern.tech/Mender/go-lib-micro/-/pipelines/348228492)

Reply:
ok then. this is the first time I see this line.